packages/core/bootstrap/src/lib/cache-warmer/actions.ts

```diff
@@ -90,6 +91,7 @@ interface WarmupFailedPayload {
   /**
    * State lookup key
    */
+  id: string
```

Comment:
How is the `id` different than `key`?

Reply:
Key is the key into the redux store for that subscription.
ID is the label of the feed. So like `BTC/ETH`. Should we call it something else?

Reply:
Changed name to `feedLabel`